problem_builder/slider.py

```diff
@@ -172,3 +172,10 @@ def validate_field_data(self, validation, data):
         Validate this block's field data.
         """
         super(SliderBlock, self).validate_field_data(validation, data)
+
+    def student_view_data(self):
```

Comment:
@pomegranited Since `SliderBlock` isn't an eligible block type, I'm wondering why it would need to define `student_view_data`?